doc/sphinxman/source/basissets_tables.rst

```diff
@@ -301,6 +302,22 @@ Appendix :ref:`apdx:basisElement`.
 |
 |
 
+.. _`table:basisPeterson`:
+
+.. table:: Summary of (other) Peterson orbital basis sets available in |PSIfour|
```

Comment:
It's not Peterson, it's Petersson

Reply:
oh right, I had K.A Petersson in the back of my mind, but it is G.A Peterson!

Reply:
that also means the table is wrong. Since the F12 are from K.A Petersson.